.github/workflows/marketing-app-deploy.yml

```diff
@@ -131,7 +131,20 @@ jobs:
       run: |
         ruby ./deploy.rb
 
-    - name: "[Staging] Deploy Marketing App to AWS"
+    - name: "Download Built Docker Image"
+      run: |
+        docker pull ${{ needs.build-and-push-image.outputs.fullImageUri }}
+        docker create --name temp ${{ needs.build-and-push-image.outputs.fullImageUri }}
+
+    - name: "Extract Static Assets from Docker"
+      run: |
+        docker cp temp:/app/apps/marketing/.next/static/ ./static
+
+    - name: "[Test] Upload Static Assets to S3"
+      run: |
+        aws s3 cp static/ s3://cdo-marketing-static-assets-test/_next/static --recursive --quiet
```

Comment:
If we were using Terraform we could possibly specify the bucket dynamically here based on what Terraform had provisioned within AWS 😢. Do we need to update it to match the bucket naming convention in the CloudFormation template?

Reply:
Good catch updated!